docs/source/algorithms/DiscusMultipleScatteringCorrection-v1.rst

```diff
@@ -130,18 +143,222 @@ The sample shape can be specified by running the algorithms :ref:`SetSample <alg
 
 The algorithm can take a long time to run on instruments with a lot of spectra and\or a lot of bins in each spectrum. The run time can be reduced by enabling the following interpolation features:
 
-- the multiple scattering correction can be calculated on a subset of the wavelength bins in the input workspace by specifying a non-default value for NumberOfWavelengthPoints. The other wavelength points will be calculated by interpolation
+- the multiple scattering correction can be calculated on a subset of the bins in the input workspace by specifying a non-default value for NumberOfSimulationPoints. The other points will be calculated by interpolation
 - the algorithm can be performed on a subset of the detectors by setting SparseInstrument=True
 
 Both of these interpolation features are described further in the documentation for the :ref:`MonteCarloAbsorption <algm-MonteCarloAbsorption>` algorithm
 
+Usage
+-----
+
+**Example - elastic calculation on single spike S(Q) and an isotropic S(Q) for comparison**
+
+.. plot::
+   :include-source:
+
+   # import mantid algorithms, numpy and matplotlib
+   from mantid import mtd
+   from mantid.simpleapi import *
+   import matplotlib.pyplot as plt
+   import numpy as np
+
+   # S(Q) consisting of single spike at q=1
+   # Spike height gives same normalisation as isotropic (integral of Q.S(Q) the same)
+   X=[0.99,1.0,1.01]
+   Y=[0.,100,0.]
+   Sofq=CreateWorkspace(DataX=X,DataY=Y,UnitX="MomentumTransfer")
+
+   # Isotropic S(Q)
+   X=[1.0]
+   Y=[1.0]
+   Sofq_isotropic=CreateWorkspace(DataX=X,DataY=Y,UnitX="MomentumTransfer")
+
+   two_thetas=[]
+   for i in range(180):
```

Comment:
Should this be 0-179 or 0-180? I think there might be an off-by-one error in this example?

Reply:
Yes having the range as 0-180 might have been nicer but I think it needs to be 0-179 because in the call to CreateSampleWorkspace I rely on the fact that the number of detectors is divisible by 4. So I need 180 detectors rather than 181.

CreateSampleWorkspace has a validation on the NumBanks parameter that means it has to be <=100 so I couldn't just supply 180 (or 181) in this parameter. So I divide by four and make each bank contain 4 detectors.